### DISCUSSION TITLE
How to serve without bundling

### DISCUSSION THREAD

**Discussion by Himavanth** [parcel-bundler/parcel#6135] [Q&A]
I am trying to serve using Parcel V2 and let it use the existing dist folder without bundling again. This is what i'm using but it just keeps updating the dist.
```
    let parcel = new Parcel({
        entries: entryFile,
        distDir: path.join(__dirname, "dist"),
        defaultConfig: require.resolve("@parcel/config-default"),
        minify: false,
        serveOptions: {
            distDir: path.join(__dirname, "dist"),
            port: 1234,
            https: false
        },        
        additionalReporters: [
            { packageName: '@parcel/reporter-cli', resolveFrom: __filename },
            { packageName: '@parcel/reporter-dev-server', resolveFrom: __filename }
        ]
    });

    await parcel.watch(err => {
        if (err) throw err;
    });
```
Any ideas ?

**Comment by mischnic** [comment]
You mean you are expecting Parcel to not rebuild because you didn't make any changes and everything should be cached?

**Reply by Himavanth** [reply]
Yes @mischnic

**Reply by Himavanth** [reply]
Looks like `parcel.watch()` does not honor the `distDir` provided in `serveOptions`. Could be a bug related to the one here https://github.com/parcel-bundler/parcel/issues/5791 ?
